fe1-web/src/features/evoting/components/ElectionOpened.tsx

```diff
@@ -159,19 +162,25 @@ const ElectionOpened = ({ election }: IPropTypes) => {
                 listStyle.push(List.hiddenItem);
               }
 
+              const onPress = () => {
+                setSelectedBallots({
+                  ...selectedBallots,
+                  [questionIndex]: ballotOptionIndex,
+                });
+              };
+
               return (
-                <ListItem key={ballotOption} containerStyle={listStyle} style={listStyle}>
+                <ListItem
+                  key={ballotOption}
+                  containerStyle={listStyle}
+                  style={listStyle}
+                  onPress={onPress}>
                   <View style={List.icon}>
                     <ListItem.CheckBox
                       testID={`questions_${questionIndex}_ballots_option_${ballotOptionIndex}_checkbox`}
                       size={Icon.size}
                       checked={selectedBallots[questionIndex] === ballotOptionIndex}
-                      onPress={() =>
-                        setSelectedBallots({
-                          ...selectedBallots,
-                          [questionIndex]: ballotOptionIndex,
-                        })
-                      }
+                      onPress={onPress}
```

Comment:
Is this still required if we already have the `onPress` on the wrapping `ListItem`?

Reply:
I thought the same but if I delete it, clicking on the checkbox wouldn't do anything. At least now you have the choice to do both!

Reply:
Huh, weird 😂👍